api/v1/air_force/views.py

```diff
@@ -19,6 +21,43 @@ def get_plane(plane_id):
     return jsonify(plane_schema.dump(plane))
 
 
+@air_force.route("/newplane", methods=["POST"])
+def put_plane():
+    name = request.json["name"]
+    size = request.json["size"]
+    speed = request.json["speed"]
+    health = request.json["health"]
+    direct_of_plane = request.json["direct_of_plane"]
+    coor_x = request.json["coor_x"]
+    coor_y = request.json["coor_y"]
+    p = add_plane(name, size, speed, health, direct_of_plane, coor_x, coor_y)
+    return jsonify(plane_schema.dump(p))
+
+
+@air_force.route("/updatedirection", methods=["PUT"])
+def update_direction():
+    id_plane = request.json["id"]
+    new_direction = request.json["direct_of_plane"]
+    old_direction = Plane.query.filter_by(id=id_plane).first().direct_of_plane
+    if (
```

Comment:
Use number to represent course

Reply:
We can use a formula for check the change of course of the plane, if the difference is grether than 2 is illegal